assets/model_monitoring/components/src/feature_importance_metrics/compute_feature_importance.py

```diff
@@ -16,10 +16,7 @@
 from shared_utilities.momo_exceptions import InvalidInputError
 from shared_utilities import constants
 from sklearn.model_selection import train_test_split
-from responsibleai import RAIInsights, FeatureMetadata
```

Comment:
we have such function which manipulates datatype specific to lightgbm. Do we still need such logic or we need some other changes?  https://github.com/Azure/azureml-assets/blob/40f26e2caf91677621af794ebb14f54a238a2979/assets/model_monitoring/components/src/feature_importance_metrics/feature_importance_utilities.py#L40

Reply:
yes we still need this I believe, since we still train a lightgbm model on the dataset